lib/src/mockserver/server.ts

```diff
@@ -24,6 +27,20 @@ export function runMockServer(
   app.use((req, resp) => {
     for (const endpoint of api.endpoints) {
       if (isRequestForEndpoint(req, pathPrefix, endpoint)) {
+        const shouldProxy = !endpoint.isDraft;
+
+        // non-draft end points get real response
+        if (shouldProxy && proxyBaseUrl) {
+          proxyRequest({
```

Comment:
For conciseness, consider:
```suggestion
          return proxyRequest({
```